tests/installation/start_install_via_yui.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
```

Comment:
the name of the the test should not contain references to libyui or be attached to UI controllers. We need something like these 2 steps:
- /installation_settings/install_installation
- /installation_settings/confirm_installation
We will be creating other test inside that intermediate folder /installation_settings/ which corresponds with the screen